core/commonMain/src/implementations/persistentOrderedMap/PersistentOrderedMap.kt

```diff
@@ -9,8 +9,9 @@ import kotlinx.collections.immutable.ImmutableCollection
 import kotlinx.collections.immutable.ImmutableSet
 import kotlinx.collections.immutable.PersistentMap
 import kotlinx.collections.immutable.implementations.immutableMap.PersistentHashMap
+import kotlinx.collections.immutable.implementations.immutableMap.PersistentHashMap.Companion.equals
```

Comment:
Not sure why this is necessary

Reply:
This is some new IDEA quirk, I have no idea (pun not intended) why this is added or even why one would import a companion's method